core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/RDFJSONWriterSettings.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Um, no, sorry, it's "Copyright (c) 2021 Eclipse RDF4J contributors." 

Remove the bit about Aduna and others - that's only used in legacy code :)

Reply:
See also https://github.com/eclipse/rdf4j/blob/main/.github/CONTRIBUTING.md#code-formatting